doc/migrate-to-motor-3.rst

```diff
@@ -0,0 +1,456 @@
+| ssl_match_hostname | tlsAllowInvalidHostnames      | ``ssl_match_hostname=True`` is equivalent to           |
+|                    |                               | ``tlsAllowInvalidHostnames=False`` and vice-versa.     |
++--------------------+-------------------------------+--------------------------------------------------------+
+| ssl_cert_reqs      | tlsAllowInvalidCertificates   | Instead of ``ssl.CERT_NONE``, ``ssl.CERT_OPTIONAL``    |
+|                    |                               | and ``ssl.CERT_REQUIRED``, the new option expects      |
+|                    |                               | a boolean value - ``True`` is equivalent to            |
+|                    |                               | ``ssl.CERT_NONE``, while ``False`` is equivalent to    |
+|                    |                               | ``ssl.CERT_REQUIRED``.                                 |
++--------------------+-------------------------------+--------------------------------------------------------+
+| ssl_certfile       | tlsCertificateKeyFile         | Instead of using ``ssl_certfile`` and ``ssl_keyfile``  |
+|                    |                               | to specify the certificate and private key files       |
++--------------------+                               | respectively,  use ``tlsCertificateKeyFile`` to pass   |
+| ssl_keyfile        |                               | a single file containing both the client certificate   |
+|                    |                               | and the private key.                                   |
++--------------------+-------------------------------+--------------------------------------------------------+
+| j                  | journal                       | -                                                      |
++--------------------+-------------------------------+--------------------------------------------------------+
+| wtimeout           | wTimeoutMS                    | -                                                      |
++--------------------+-------------------------------+--------------------------------------------------------+
+
+MotorClient.fsync is removed
+............................
+
+Removed :meth:`~motor.motor_tornado.MotorClient.fsync`. Run the
+`fsync command`_ directly with :meth:`~motor.motor_tornado.MotorDatabase.command`
+instead. For example::
+
+    await client.admin.command('fsync', lock=True)
+
+.. _fsync command: https://mongodb.com/docs/manual/reference/command/fsync/
+
+MotorClient.unlock is removed
+.............................
+
+Removed :meth:`~motor.motor_tornado.MotorClient.unlock`. Run the
+`fsyncUnlock command`_ directly with
+:meth:`~motor.motor_tornado.MotorDatabase.command` instead. For example::
+
+     await client.admin.command('fsyncUnlock')
+
+.. _fsyncUnlock command: https://mongodb.com/docs/manual/reference/command/fsyncUnlock/
+
+MotorClient.is_locked is removed
```

Comment:
Motor 2 doesn't support is_locked so we can remove this one.

Reply:
Done